src/NzbDrone.Automation.Test/PageModel/PageBase.cs

```diff
@@ -27,6 +27,11 @@ public IWebElement Find(By by, int timeout = 5)
             return wait.Until(d => d.FindElement(by));
         }
 
+        public bool Displayed(By by)
+        {
+            return _driver.FindElement(by).Displayed;
```

Comment:
Added this so I can assert the actual exception rather than the wait timeout exception from the above wrapper methods.